contingency-api/src/test/java/eu/itesla_project/contingency/tasks/BranchTrippingTest.java

```diff
@@ -84,4 +88,21 @@ public void unknownSubstationTrippingTest() {
         BranchTripping tripping = new BranchTripping("NHV2_NLOAD", "UNKNOWN");
         tripping.modify(network, null);
     }
+
+    @Test
+    public void fictitiousSwitchTest() {
+        Network network = FictitiousSwitchFactory.create();
+
+        BranchTripping tripping = new BranchTripping("CJ", "C");
+
+        Set<Switch> switchesToOpen = new HashSet<>();
+        Set<Terminal> terminalsToDisconnect = new HashSet<>();
+        tripping.traverse(network, null, switchesToOpen, terminalsToDisconnect);
+        assertEquals(Sets.newHashSet("BD", "BL"), switchesToOpen.stream().map(Switch::getId).collect(Collectors.toSet()));
+        assertEquals(Collections.emptySet(), terminalsToDisconnect);
+
+        tripping.modify(network, null);
+        assertTrue(network.getSwitch("BD").isOpen());
```

Comment:
Maybe it would be more robust to assert not only open switches but the position of all of them?